package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "eslint ./ --ext .ts",
     "lint:fix": "eslint ./ --ext .ts --fix",
     "db:migrate:create": "ts-node ./node_modules/.bin/typeorm migration:create -n",
-    "db:migrate": "ts-node ./node_modules/.bin/typeorm migration:run",
+    "db:migrate": "ts-node ./node_modules/typeorm/cli.js migration:run",
     "db:rollback": "ts-node ./node_modules/.bin/typeorm migration:revert",
     "db:seed:create": "ts-node ./node_modules/.bin/typeorm migration:create -c seed -n",
     "db:seed": "ts-node ./node_modules/.bin/typeorm migration:run -c seed",
```

Comment:
can you also change these to use `./node_modules/typeorm/cli.js` so that they would work Windows 10 machines? The new path seems to work on Windows + WSL (and thus Linux), and most likely on Mac.